ProjekatWebPortal/wwwroot/Content/Konfiguracija/MojaKonfiguracija.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Molim te obriši ovu konfiguraciju. Ona je ranije služila za vesti kako bi se glavna vest sačuvala negde. Sada to držimo u tabeli, tako da je ovo nepotrebno